blackbox/docs/config/logging.rst

```diff
@@ -40,114 +63,132 @@ configuration.
 And here is a snippet of the generated properties ready for use with log4j.
 You get the point.
 
-.. code-block:: yaml
+.. SEEALSO::
 
-  log4j.rootLogger=INFO, console
+    Consult the `PropertyConfigurator`_ documentation or the `configuration
+    section`_ of the Log4j documentation for more information.
 
-  log4j.logger.action=DEBUG
+.. _conf-logging-log4j-loggers:
 
-  log4j.appender.console=org.elasticsearch.common.logging.log4j.ConsoleAppender
-  log4j.appender.console.layout=org.apache.log4j.PatternLayout
-  log4j.appender.console.layout.conversionPattern=[%d{ISO8601}][%-5p][%-25c] %m%n
+Loggers
+.......
 
-   ...
+.. FIXME: need a full list of available loggers
 
-.. _PropertyConfigurator: https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PropertyConfigurator.html
+Possible log levels are the same as for Log4j_: ``TRACE``, ``DEBUG``, ``INFO``,
+``WARN``, and ``ERROR``. They must be provided as string literals in the
+``SET`` statement.
+
+.. NOTE::
+
+   Be careful using the ``TRACE`` log level because it's extremely verbose,
+   can obscure other important log messages and even fill up entire data disks
+   in some cases.
 
-Settings
---------
+.. FIXME: how to configure root logger?
```

Comment:
@mxm can you help me here

we mention how do this at run-time and I would like to mention it here too if it makes sense

Reply:
It's described in this document how to configure the log level of the root logger. Would be ok to mention it here.

Reply:
hmm so just `logger._root`? because the log4j docs indicate that you might not be able to address the root logger via this method but I might be misunderstanding

Reply:
I understood that this is just our way to address the root logger via the `SET` statement at runtime. How this is internally mapped to log4j calls might be different.